tests/tests_reboot.yml

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Ensure we can set crypto policy including reboot of the system
+  hosts: all
+  roles:
+    - role: linux-system-roles.crypto_policies
+      vars:
+        crypto_policies_policy: "FUTURE"
+
+  tasks:
+    - name: Verify the facts are correctly set
+      block:
+        - meta: flush_handlers
+
+        - name: Check the current policy is FUTURE
+          assert:
+            that:
+              - "crypto_policies_active == 'FUTURE'"
+              - "crypto_policies_reboot_required == false"
```

Comment:
`- not crypto_policies_reboot_required`

Reply:
also - don't need to use double quotes for the first condition

Reply:
Fixed. Thanks